frontend/src/components/elements/ImageList/ImageList.tsx

```diff
@@ -64,6 +64,9 @@ class ImageList extends React.PureComponent<Props> {
       throw Error(`Invalid image width: ${protoWidth}`)
     }
 
+    console.log("===")
+    console.log(width)
+    console.log(imgWidth)
```

Comment:
Remove logging before merging PR.

Reply:
Are images responding to full screen width or height?

Reply:
the problem is: you can set the width in your script.
$ st.image(channels, caption=["Red", "Green", "Blue"], width=200)

And, I don't know if make sense expand this component...